database/prod/queryClauses/whereClause.cpp

```diff
@@ -0,0 +1,47 @@
+#include "whereClause.h"
+
+WhereStatement::~WhereStatement() {}
```

Comment:
если в конструкторах со списком инициализации ешё можно подумать ад скобками, то вот здесь без их переноса вообще теряется что это функция.

```suggestion
WhereStatement::~WhereStatement() {
}
```